.github/workflows/wheels.yml

```diff
@@ -0,0 +1,123 @@
+          echo github.ref ${{ github.ref }}
+
+  build_wheels:
+    name: Build wheel for ${{ matrix.python }}-${{ matrix.buildplat[1] }} ${{ matrix.buildplat[2] }}
+    # needs: get_commit_message
+    #    if: >-
+    #      contains(needs.get_commit_message.outputs.message, '[wheel build]') ||
+    #      github.event_name == 'schedule' ||
+    #      github.event_name == 'workflow_dispatch' ||
+    #      (github.event_name == 'pull_request' &&
+    #      (contains(github.event.pull_request.labels.*.name, '36 - Build') ||
+    #      contains(github.event.pull_request.labels.*.name, '14 - Release'))) ||
+    #      (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') && ( ! endsWith(github.ref, 'dev0')))
+    runs-on: ${{ matrix.buildplat[0] }}
+    strategy:
+      # Ensure that a wheel builder finishes even if another fails
+      fail-fast: false
+      matrix:
+        # Github Actions doesn't support pairing matrix values together, let's improvise
+        # https://github.com/github/feedback/discussions/7835#discussioncomment-1769026
+        buildplat:
+        # - [ubuntu-20.04, manylinux*, x86_64]
+        # - [ubuntu-20.04, manylinux*, aarch64]`
+        # - [macos-11, macosx*, x86_64]
+        - [windows-2019, win*, AMD64]
+        # - [windows-2019, win32]
+        python: ["cp38"] #, "cp39", "cp310", "cp38", "cp310", "pp38" "pp39"]
+      #    env:
+      #      IS_32_BIT: ${{ matrix.buildplat[1] == 'win32' }}
+      #      IS_PUSH: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') }}
+      #      IS_SCHEDULE_DISPATCH: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
+    steps:
+      - name: Checkout scipy
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      # Used to push the built wheels
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
+
+      - name: install-rtools
+        run: |
+          # provides pkg-config
+          choco install rtools --no-progress
+          echo "c:\rtools40\ucrt64\bin;" >> $env:GITHUB_PATH
+        if: runner.os == 'Windows'
+
+      #      - name: Configure mingw for 32-bit builds
+      #        run: |
+      #          # Force 32-bit mingw
+      #          choco uninstall mingw
+      #          choco install -y mingw --forcex86 --force --version=7.3.0
+      #          echo "C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw32\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+      #          refreshenv
+      #        if: ${{ env.IS_32_BIT == 'true' }}
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.4.0
+        env:
+          CIBW_BUILD: ${{ matrix.python }}-${{ matrix.buildplat[1] }}
+          CIBW_ARCHS: ${{ matrix.buildplat[2] }}
+          CIBW_ENVIRONMENT: PKG_CONFIG_PATH=c:\\opt\\openblas\\if_32\\64\\lib\\pkgconfig
```

Comment:
[The example](https://cibuildwheel.readthedocs.io/en/stable/options/#examples_6) uses quotes. Also, it might be better to be specific to windows with `CIBW_ENVIRONMENT_WINDOWS`